SDK/CalendarStoreViewController.swift

```diff
@@ -63,9 +65,9 @@ public final class CalendarStoreViewController: UITabBarController {
      - parameter largeTitle: Set to `false` if you don't want to use large navigation bar titles.
      - parameter tintColor: The tint color used through out the SDK, default is SchedJoules red.
      */
-    private init(apiKey: String, pageIdentifier: String?, title: String?, largeTitle: Bool = true, tintColor: UIColor = ColorPalette.red) {
+    public init(apiKey: String, pageIdentifier: String?, title: String?, largeTitle: Bool = true, tintColor: UIColor = ColorPalette.red) {
         // Initialization
-        self.apiKey = apiKey
+        self.apiClient = SchedJoulesApi(accessToken: apiKey)
```

Comment:
apiClient should be a parameter of the primary ctor. Only secondary ctors should take the apiKey. That way the primary ctor does't contain any code and you can easily mock/replace the API implementation.

Reply:
Fixed.